sys/net/network_layer/fib/fib.c

```diff
@@ -496,21 +547,23 @@ int fib_get_destination_set(uint8_t *prefix, size_t prefix_size,
 
     for (size_t i = 0; i < FIB_MAX_FIB_TABLE_ENTRIES; ++i) {
         if ((fib_table[i].global != NULL) &&
-            (universal_address_compare_prefix(fib_table[i].global, prefix, prefix_size<<3) >= 0)) {
-            if( (dst_set != NULL) && (found_entries < *dst_set_size) ) {
-            /* set the size to full byte usage */
-            dst_set[found_entries].dest_size = sizeof(dst_set[found_entries].dest);
-            universal_address_get_address(fib_table[i].global,
-                                          dst_set[found_entries].dest,
-                                          &dst_set[found_entries].dest_size);
+            (universal_address_compare_prefix(fib_table[i].global, prefix,
+                                              prefix_size << 3) >= 0)) {
```

Comment:
very nit-picking here: you are using the shift by 3 quite often throughout  the code to get the amount of bits. I did invest some time to understand the shift at first. A macro like `#define IN_BITS(X) (X << 3)` would be more readable. (: